docker/Dockerfile

```diff
@@ -48,12 +45,11 @@ RUN Rscript -e  "install.packages( \
       'rprojroot', \
       'viridis', \
       'spelling', \
-      'styler', \
-      'WGCNA'))"
+      'styler'))"
 
 # Install bioconductor packages
 # org.Mm.eg.db and org.Dr.eg.db are required for gene mapping
-RUN R -e "options(warn = 2); BiocManager::install( \
+RUN Rscript -e "options(warn = 2); BiocManager::install( \
```

Comment:
Does it not matter if impute is not installed before WGCNA? WGCNA needs it to install. That's why I put it there.